runtime/src/main/java/org/corfudb/protocols/wireprotocol/TokenRequest.java

```diff
@@ -39,7 +39,7 @@
     final Long numTokens;
 
     /** The streams which are written to by this token request. */
-    final Set<UUID> streams;
+    final List<UUID> streams;
```

Comment:
Just curiosity: Did we see a perf gain with List vs Set. Or is it for more because it is more intuitive to use list for this.

Reply:
a set doesnt denote order. I want to send an ordered container so that the response doesn't have to contain the the stream ids.

Reply:
In case, avoiding duplicates matters LinkedHashSet could also be considered as it iterates predictably with insertion order.

Reply:
I don;t think its generic enough to expose in an interface. I will create newer apis in the sequencer view, to hide these details from the interface. For example, queryGlobalTail/queryStreamTail/queryStreamTails